### ISSUE TITLE
Catch-all mailbox

### ISSUE THREAD

**Issue by fo-fo** [inbucket/inbucket#199]
I have recently been trying Inbucket and it seems like a great project!

However, I was wondering if it would be possible/feasible to forward all received messages into the same mailbox? I would like to use the POP3 feature to get a notification from a mail client when a message is sent in my development environment, but there are so many recipient addresses that adding an email account in the mail client for all those addresses would be a lot of work.

I can imagine 3 ways to implement this:
1. A built in "catch-all" mailbox that receives a copy of each message.
2. A new "mailbox naming" setting (`static`?) that ignores the recipient email address in mailbox selection and uses a static mailbox name instead.
3. A "virtual" catch-all mailbox that holds nothing but returns the results from all other mailboxes when queried.

Is this feasible? I may be able to make a pull request if it's something that could be done with a reasonable amount of effort.

**Comment by jhillyerd** [comment]
Inbucket doesn't handle single large mailboxes well, so a feature like this may not be great for busier environments.   The monitor tab was my way of solving this problem, but obviously it doesn't send alerts.

For your numbered options:

1. I wouldn't want to leave this on by default, given the performance issues that could arise.
2. This seems like the best fit, since it's opt-in.
3. This would be very difficult to implement with Inbuckets current design, it doesn't have a central DB of all mail.

**Comment by fo-fo** [comment]
I guess a 4th option would be to use the [Notification API](https://developer.mozilla.org/en-US/docs/Web/API/Notification) to display a desktop notification from the monitor when a new message arrives.

**Comment by jhillyerd** [comment]
While I like the idea, all the major browsers require HTTPS for notifications, which is likely to be a significant hurdle for many users.

**Comment by fo-fo** [comment]
> While I like the idea, all the major browsers require HTTPS for notifications, which is likely to be a significant hurdle for many users.

Yeah. Then again, it seems that most browser also consider localhost/127.0.0.1 as a secure origin. At least for my use case this would be sufficient (although I haven't personally tested how much variation there is in browser support.)
